docs/api-reference/requests/README.md

```diff
@@ -0,0 +1,112 @@
+# Requests
```

Comment:
Here, I think plural works, because there indeed many different kinds of requests. There are subpages to this page